### ISSUE TITLE
Upload plugin to conda-forge

### ISSUE THREAD

**Issue by Czaki** [napari/midi-app-controller#108]
We should provide conda-forge version of this plugin

**Comment by jni** [comment]
Indeed! Thanks @Czaki. I'd like to get #111 merged and then we can make a release and make the PR to conda-forge/staged-recipes.

**Comment by Czaki** [comment]
ouch

**Comment by willingc** [comment]
I wonder if using https://github.com/PortMidi/portmidi would unblock this.
